develop/application-devs/interact/understand-parachain-data.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Understanding Metadata
+description: Learn how to interact and decipher parachain data (metadata).
+---
+
+Each Polkadot SDK-based network exposes information about its runtime, allowing developers to know granular details regarding the types, pallets, and runtime APIs that a chain may offer.  Metadata files are SCALE-encoded, and allow the development of browser-based, mobile, or desktop applications that can keep up with potential forkless upgrades.  The concept of metadata also aids in ensuring that a single application can support many Polkadot SDK-based chains at once.
+
+## Exposing runtime information as metadata
+
+To interact with a node or the information stored in the blockchain, you need to know how to connect to the chain and how to access the features the runtime exposes to the outside world.
+In general, this interaction involves a remote procedure call (RPC) to request information you're interested in retrieving or updating.
+An application developer typically needs to know about the contents of the runtime's logic, including the following details:
```

Comment:
Would be nice if we could reach out to developers like Ross and create a concise list of what he typically looked for while developing the staking dashboard.